script.js

```diff
@@ -1,69 +1,63 @@
 const cardContainer = document.querySelector(".card-container");
-const newGameButton = document.getElementById("new-game-btn");
+const alertContainer = document.querySelector("[data-id='alert-container']");
+const alertOverlay = document.querySelector("[data-id='alert-overlay']");
+const winTime = document.querySelector("[data-id='win-time']");
+const winMovements = document.querySelector("[data-id='win-movements']");
+const newGameButtons = document.querySelectorAll("[data-id='new-game-btn']");
 // timer
 const seconds = document.getElementById("seconds");
 // movements
 const movementsContainer = document.getElementById("movements");
-
-const cards = [
-  {
-    name: "bear",
-    img: "./assets/images/cards/bear.svg",
-  },
-  {
-    name: "crab",
-    img: "./assets/images/cards/crab.svg",
-  },
-  {
-    name: "shrimp",
-    img: "./assets/images/cards/shrimp.svg",
-  },
-  {
-    name: "lion",
-    img: "./assets/images/cards/lion.svg",
-  },
-  {
-    name: "zebra",
-    img: "./assets/images/cards/zebra.svg",
-  },
-  {
-    name: "fox",
-    img: "./assets/images/cards/fox.svg",
-  },
-  {
-    name: "jellyfish",
-    img: "./assets/images/cards/jellyfish.svg",
-  },
-  {
-    name: "penguin",
-    img: "./assets/images/cards/penguin.svg",
-  },
-];
-
+// fetch Data----------------------------------------------
+
+function getData() {
+  fetch(
+    "https://raw.githubusercontent.com/Niloufar97/Niloufar97.github.io/main/memory-game/cards.json"
+  )
+    .then((response) => response.json())
+    .then((cards) => {
+      updateCards(cards);
+    });
+}
 let selectedCards = [];
 const matchedCards = [];
 let clickCounter = 0;
 let movementsCounter = 0;
 let timer = null;
+let timeoutId = null;
 
-// doublecards----------------------------------------------------------------------
+// rendering cards-----------------------------------------------------------------
 
-const doublingCards = () => {
-  let doubleCards = [...cards, ...cards];
-  const doubleCardsWithId = doubleCards.map((card, index) => {
-    return { ...card, id: index + 1 };
-  });
-  return doubleCardsWithId;
+const updateCards = (cards) => {
+  const shuffledcards = shufflingCards(cards);
+    shuffledcards.forEach((card) => {
+      const cardDiv = createCardDiv(card);
+      cardContainer.appendChild(cardDiv);
+      cardDiv.addEventListener("click", () => {
+        clickHandler(cardDiv, card);
+      });
+    });
 };
 
 //shuffle cards------------------------------------------------------------
 
-const shufflingCards = () => {
-  const cards = doublingCards();
-  const shuffledCards = cards.sort(() => Math.random() - 0.5);
+const shufflingCards = (cards) => {
+  const doubledCards = doublingCards(cards);
```

Comment:
For the sake of separation of concerns and cleanliness, I would recommend to not double the cards here, as the function does not seem like it should double anything (it should only be concerned with shuffling). 
It would be better to call the respective functions all from the "updateCards" function. 🙏🏽 
A minor thing, really, but good to get good habits :)